pkg/api/types.go

```diff
@@ -1664,6 +1667,104 @@ type SecretList struct {
 	Items []Secret `json:"items"`
 }
 
+// SecurityContext holds security configuration that will be applied to a container.  If a security context
+// is set on the container spec then it must comply with any constraints defined in the SecurityConstraints context
```

Comment:
Apologies, poor pronoun use.   we := myself and @smarterclayton discussed in the link provided above.